package.json

```diff
@@ -4,6 +4,11 @@
   "private": false,
   "homepage": "https://linkfree.eddiehub.org",
   "dependencies": {
+    "@emotion/react": "^11.10.4",
```

Comment:
We are using prime react for the current version of linkfree. Please try to look at their implementation of dark mode instead of installing lots of new packages